wp-remember-the-galleries/wp-remember-the-galleries.js

```diff
@@ -320,25 +320,37 @@
 				else {
 					this.toolbar.set(new media.view.Toolbar({ controller: this }));
 				}
+			},
 
-				this.saveGalleryButton = new media.view.Button({
-					click: function() {
-						this.controller.saveGallery();
-					},
-					disabled: true,
-					style: 'primary',
-					text: "Save Gallery",
-					controller: this,
-				});
+			updateButtonState: function() {
+				var toolbar = this.toolbar.get();
+
+				if(toolbar) {
+					this.saveGalleryButton = new media.view.Button({
+						click: function() {
+							this.controller.saveGallery();
+						},
+						disabled: true,
+						style: 'primary',
+						text: "Save Gallery",
+						controller: this,
+					});
 
-				this.toolbar.get().unset('insert');
 
-				this.toolbar.get().set({
-					'save-gallery': this.saveGalleryButton
-				});
-			},
+					toolbar.unset('insert');
+
+					if(wp.media.frame.galleryDetails.get('id')) {
```

Comment:
Are my eyes deceiving me or do both routes of this `if` block do the exact same thing?